reporter/src/funTest/kotlin/reporters/EvaluatedModelReporterTest.kt

```diff
@@ -22,10 +22,9 @@ package org.ossreviewtoolkit.reporter.reporters
 import io.kotest.core.spec.style.WordSpec
 import io.kotest.matchers.shouldBe
 
-import org.ossreviewtoolkit.model.OrtIssue
-
 import java.io.File
 
+import org.ossreviewtoolkit.model.OrtIssue
```

Comment:
That change belongs to the previous commit.

Reply:
Done.